### PR TITLE
Added version endpoint

### LINKED ISSUE #10: Version endpoint
Follow example from https://github.com/iver-wharf/wharf-core/pull/2 once it has been merged

### PR DESCRIPTION
**Pull request by jilleJr**
- Added `version.go` with `GET /import/github/version` endpoint
- Added `deploy/update-version.sh` to update `version.yaml` on Docker build
- Updated main.go with metadata
- Updated .wharf-ci.yml to populate `BUILD_*` build args
- Updated `CHANGELOG.md`
- Updated `swag init` docs & usage as we're now relying on types from foreign packages (`app.Version`)

Based on https://github.com/iver-wharf/wharf-core/pull/2

Screenshot_2021-05-28_17-06-59

Closes #10

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -13,21 +13,28 @@ import (
 	"github.com/gin-contrib/cors"
 	"github.com/gin-gonic/gin"
 
-	ginSwagger "github.com/swaggo/gin-swagger"
-	"github.com/swaggo/gin-swagger/swaggerFiles"
-	_ "github.com/iver-wharf/wharf-api/docs"
+	"github.com/iver-wharf/wharf-api/docs"
```
Comment (fredx30) [inline main.go:16]: This path does not appear to resolve.

**Review by Pikabanga — COMMENTED** [review]
Same comments as in https://github.com/iver-wharf/wharf-provider-github/pull/5

**Review by Pikabanga — CHANGES_REQUESTED** [review]
Flagging for requesting changes

File: main.go (hunk 1 above)
Comment (jilleJr) [reply main.go:16]: It is generated by `swag init`

**Review by fredx30 — APPROVED** [review]

**Review by Pikabanga — APPROVED** [review]

**Comment by jilleJr** [comment]
Oh gosh I merged before having 2 reviewers...

Got jinxed by `gh` CLI tool

```console
$ gh pr status
Relevant pull requests in iver-wharf/wharf-api

Current branch
  #2  Added version endpoint [feature/version-endpoint]
  ✓ Approved ✓ Up to date
```

It did say approved. :( Will check closer next time

**Pull request merged**